Gemfile.lock

```diff
@@ -2,35 +2,35 @@ PATH
   remote: .
```

Comment:
Per our discussion, remove `Gemfile.lock`. We should also add it to `.gitignore`.